build/documentation/copy-package-readme.js

```diff
@@ -29,36 +31,130 @@ process.argv.forEach(function (val, index) {
         console.log('Invalid argument used. To perform a dry-run use --dry-run');
         process.exit(1);
     }
+
 });
 
 /**
- * Function to copy readme files to the docs/en/packages folder
+ * Run script
+ * Based off presence of --dry-run parameter
  */
-function copyReadmeFiles() {
+dryRun? runDryRun() : copyReadmeFiles();
+
+
+/**
+ * Function that describes what this script will do and
+ * where, without making any actual changes
+ */
+function runDryRun() {
 
     const resolvedSrcReadmePaths = path.resolve(rootDir, srcReadmePaths);
 
-    glob(resolvedSrcReadmePaths, {realpath:true}, function(error, files) {
-        
-        files.forEach((filePath) => {
-            const destReadmePath = filePath.replace(/(\bpackages\b)(?!.*\1)/, destDir);
+    console.log("   In docs/en/packages/, this script would...");
+
+    glob(resolvedSrcReadmePaths, {realpath:true}, function(error, srcFiles) {
+        srcFiles.forEach((srcReadmePath) => {    
+
+            const destReadmePath = srcReadmePath.replace(/(\bpackages\b)(?!.*\1)/, destDir);
             const dirPath = path.dirname(destReadmePath);
+            const srcDirPath = path.dirname(srcReadmePath);
+        
+            var title = srcDirPath
+                .split(path.sep)
+                .pop()
+        
+            var msg = '';
+
             if (!fs.existsSync(dirPath)) {
-                dryRun ? console.log("----> Would create folder " + dirPath) : fs.mkdirSync(dirPath);
+                msg += "      ...CREATE folder '" + title + "'\n";
             }
-            if (!fs.existsSync(destReadmePath)) {
-                dryRun ? console.log("  --> Would create file README.md in " + dirPath) : fs.copyFileSync(filePath, destReadmePath);
-            } else {
-                dryRun ? console.log("  --> Would replace README.md in " + dirPath) : fs.copyFileSync(filePath, destReadmePath);
+            if (!fs.existsSync(destReadmePath)) {
+                msg += "       ...ADD readme.md into the '" + title + "' folder";
+            } else {
+                msg += "       ...REPLACE readme.md in the '" + title + "' folder";
```

Comment:
I would prefer backticks to be used instead of concat +, that would make this something like:

\`       ...REPLACE readme.md in the '${title}' folder\`